packages/scandipwa/src/component/CheckoutOrderSummary/CheckoutOrderSummary.style.scss

```diff
@@ -19,9 +19,9 @@
     @include desktop {
         border: 1px solid var(--secondary-base-color);
         border-radius: 2px;
-        padding-block-start: 24px;
-        padding-block-end: 0;
+        padding-block: 24px 0;
         padding-inline: 16px;
+        margin: 0 0 15px 0;
```

Comment:
Instead can you use only margin-block-end: 15 px? Or it is required to be all other nulls?

Reply:
Basically as i understand it should be all other nulls (if i get it right). But i don't think it is required, so will change